Solutions/Commvault Security IQ/Data/Solution_Commvault Security IQ.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Name": "Commvault Security IQ",
+  "Author": "svc.cv-securityiq@commvault.com",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/Commvault-Logo.svg\" width=\"75px\"height=\"75px\">",
+  "Description": "This Sentinel integration enables Commvault users to ingest alerts and other data into their Sentinel instance. With Analytic Rules, Sentinel can automatically create Sentinel incidents",
+  "Analytic Rules": [
+	  "Analytic Rules/Data_Alert.yaml",
+	  "Analytic Rules/IDP_Alert.yaml",
+	  "Analytic Rules/User_Alert.yaml"
+  ],
+  "Playbooks": [
+        "Playbooks/CommvaultLogicApp/azuredeploy.json",
```

Comment:
Please remove the comma at the end of this line.

Please repackage the solution after making all the suggested changes.